src/store/models/network.ts

```diff
@@ -679,9 +681,9 @@ const networkModel: NetworkModel = {
     if (networks.length === 0) {
       ipcRenderer.send('docker-shut-down');
     }
-    networks.forEach(async network => {
+    for (const network of networks) {
       await actions.stop(network.id);
-    });
+    }
```

Comment:
It was intentional to stop the networks in a `forEach` so that it would be faster by running in parallel. The intervel setup below will repeatedly check if all the networks are stopped before sending the IPC shut down message. So your change here can be reverted.

Reply:
Ohh okay. Changes Reverted